examples/flyteinteractive_plugin/requirements.txt

```diff
@@ -124,8 +124,8 @@ fastjsonschema==2.19.0
 flyteidl==1.10.6
     # via flytekit
 flytekit==1.10.2
-    # via flytekitplugins-flyin
-flytekitplugins-flyin==1.10.2
+    # via flytekitplugins-flyteinteractive
+flytekitplugins-flyteinteractive==1.10.2
```

Comment:
We should regenerate this file here since`flytekitplugins-flyteinteractive==1.10.2` doesn't exist

```
pip-compile requirements.in
```

Reply:
Fixed in 1e7d937fdf91b598900822245334869ed79979d4